src/LibraryManager.Vsix/Json/SuggestedActions/SuggestedActionProvider.cs

```diff
@@ -43,16 +43,22 @@ public IEnumerable<ISuggestedAction> GetSuggestedActions(ITextView textView, ITe
 
             yield return new UninstallSuggestedAction(this, LibraryCommandService);
 
+#pragma warning disable CA2000 // Dispose objects before losing scope
```

Comment:
I assume you're skipping dispose because it is returned... any concern about the case where it is not returned (`!update.HasActionSets`)?

Reply:
Since the Dispose implementation doesn't do anything, I'm not concerned.  Yes, we are breaching the convention, but there's no effect.  If something were to change here where Disposing were important, that could be bad, but at this point it's more likely for this all to get rewritten into an LSP-based extension (so we would no longer be inheriting IDisposable from the VS Editor's ISuggestedAction) before any other major changes would happen.